css/types/easing-function.json

```
@@ -5,7 +5,7 @@
         "__compat": {
           "description": "<code>&lt;easing-function&gt;</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/easing-function",
-          "spec_url": "https://drafts.csswg.org/css-easing/#easing-functions",
+          "spec_url": "https://drafts.csswg.org/css-easing-2/#easing-functions",
```

It looks like the linear function is also available in Level 1, so we should stick with the L1 spec for now, based upon @Elchi3's research up above!

```suggestion
          "spec_url": "https://drafts.csswg.org/css-easing/#easing-functions",
```